upp/classes/preprocessing_config.py

```diff
@@ -110,9 +110,16 @@ def __post_init__(self):
         )
 
         # copy config
-        git_hash = check_output(["git", "rev-parse", "--short", "HEAD"], cwd=Path(__file__).parent)
-        self.git_hash = git_hash.decode("ascii").strip()
-        self.config["pp_git_hash"] = self.git_hash
+        try:
+            git_hash = check_output(
+                ["git", "rev-parse", "--short", "HEAD"], cwd=Path(__file__).parent
+            )
+            self.git_hash = git_hash.decode("ascii").strip()
+            self.config["pp_git_hash"] = self.git_hash
+        except Exception:
+            log.warning("Could not get git hash")
+            self.git_hash = "?"
```

Comment:
what would be great is if we could fallback to the `__version__` if the hash is not available, since this will uniquely define the code when installing from pip. but we could open an issue and fix this also in salt by implementing int he tools package if you don't want to do it now

Reply:
https://github.com/umami-hep/atlas-ftag-tools/issues/56

Reply:
I would like to finish the integration MR in umami as soon as possible as it was hanging there for quite a while already

Reply:
Furthermore I am not sure if it is a good idea to make it into tools that would be outside of its initial scope already

Reply:
How so? I think of the tools package as a place for code that might otherwise be duplicated in other ftag repos

Reply:
I just mean from the documentation of the package it is said that its main use is to work with TDD files